smart-contracts/developing-contracts/verify-a-contract.md

```diff
@@ -25,6 +25,196 @@ The following guide walks you through the process of contract verification using
 * [Remix](https://remix.ethereum.org/)
 
 ### Procedure
+#### Via. API (Recommended)
```

Comment:
This is the same content as [Filfox docs](https://filfox.notion.site/Filfox-Contract-Verification-API-Documents-c48d361c949348acb0bf806871ddd2c2). I am wondering if this is necessary to have the same content here in Filecoin docs to bring the extra work of maintaining here comparing with just providing a link to it.

Reply:
I agree with @longfeiWan9, I think a better option would be to list the different platforms that do contract verification, like at the top of this document, and then under each platform list links to their tutorials.